src/other/python-ext/extension.cpp

```diff
@@ -43,20 +44,68 @@ static PyObject* get_block(PyObject* self, PyObject *args) {
     }
     return Py_BuildValue("s", block.Format().c_str());
 }
+
 static PyObject* get_trans(PyObject* self, PyObject *args) {
-    hash_t hash;
+  
+    CTransaction trans;
+    char* hash;
+
```

Comment:
Single blank line please. Plus, I prefer 'char *hash' to 'char* hash'. Old habbits die hard.